pkg/controller/constrainttemplate/constrainttemplate_controller_test.go

```diff
@@ -130,8 +141,7 @@ violation[{"msg": "denied!"}] {
 	// Setup the Manager and Controller.  Wrap the Controller Reconcile function so it writes each request to a
 	// channel when it is finished.
 	mgr, wm := setupManager(t)
-	c := &testclient.RetryClient{Client: mgr.GetClient()}
-	ctx := context.Background()
+	c := &testclient.RetryClient{Client: mgr.GetClient(), Limiter: rate.NewLimiter(2, 2)}
```

Comment:
* Was this meant to use the constants above?
* Would it be an easier interface if we create a `NewRetryClient` constructor that sets a default limiter?

Reply:
added `NewRetryClient` constructor with default limiter values